src/main/webapp/data/userMapData.json

```diff
@@ -1,5 +1,5 @@
 {
-  "lastUpdated": "End of September 2017",
+  "lastUpdated": "End of November 2017",
```

Comment:
Additional note: You might want to leave the date as it is (should be "October" after resolving the merge conflict). Usually we tend to keep the date to represent the date for the most recent dataset, as opposed to when it is actually put onto the site.